tutorial/setup/self-hosting.md

```diff
@@ -38,3 +39,16 @@ To run the server you now simply run `python3 planarserver.py` and your server s
 If everything went well you should now be able to visit `http://localhost:8000` and be greeted with the login screen.
 
 For more info on how to configure some other aspects of your server visit the main [server management](/docs/dm/management/) docs.
+
+## Docker Container
```

Comment:
Maybe an example about attaching volumes could be handy here as well.

Reply:
I'll be honest here and my knoweledge of docker only just came from this [video](https://youtu.be/eGz9DS-aIeY) which doesn't cover volumes. Gonna read some docs and see if I can get a running example.

Reply:
Oh my bad I thought you were using docker yourself, don't feel pressure to write about things you're not familiar with.  Leaving it like this is fine for now, me or someone else will pick it up!

Reply:
Well I am, but I'm not really using volumes. I'm just pulling the files out of the container with  ```docker cp container:/file/path/ /destination/path/```

Reply:
That said I got the [docker document](https://docs.docker.com/storage/volumes/) pulled up just making myself a cup of tea and a bologna sandwich first

Reply:
got a working example of setting up volumes